content/applications/sales/subscriptions/products.rst

```diff
@@ -94,13 +124,24 @@ the product form, select a pricelist in the :guilabel:`Pricelist` column.
 When pricelists are added to the :guilabel:`Time-based pricing` tab, the pricelist form in the
 *Sales* app is automatically updated.
 
-Time-based pricing rules can also be configured directly on the pricelist form. To do this, go to
-:menuselection:`Sales --> Products --> Pricelists` and select a pricelist (or click :guilabel:`New`
-to create a new pricelist). In the :guilabel:`Time-based rules` tab, click :guilabel:`Add a line`.
+Time-based pricing rules can also be configured directly on the pricelist form.
+
+To do this, go to :menuselection:`Sales app --> Products --> Pricelists`, and select a pricelist (or
+click :guilabel:`New` to create a new pricelist).
```

Comment:
May want to call out that this is also accessible through the Subscription app (follows the same menu steps)